pages/linux/zenity.md

```diff
@@ -0,0 +1,23 @@
+# zenity
+
+> Display dialogs from the command line/shell scripts, and return user-inserted values or 1 if error.
+
+- Display the default question dialog:
+
+`zenity --quesion`
+
+- Display an info dialog displaying the text "Hello!":
+
+`zenity --info --text="Hello!"`
+
+- Display a name/password form and output the data separated by ';':
+
+`zenity --forms --add-entry="Name" --add-password="Password" --separator=";"`
```

Comment:
Same here. Could you wrap the things that the user can change in `{{` and `}}`, please?